simtools/util/general.py

```diff
@@ -854,3 +856,41 @@ def save_dict_to_file(dictionary, file_name):
     """
     with open(file_name, "wb") as f:
         pickle.dump(dictionary, f)
+
+
+def parse_astropy_unit(unit_in_string):
```

Comment:
As you saw in Max's suggestion, this can be simplified to be just a couple of lines which should actually be put back into `_get_header_astropy_units` in `corsika_output.py`

Reply:
Indeed, I updated it with his suggestion, it is much simpler now.